parm/soca/obsprep/obsprep_config.yaml

```diff
@@ -39,3 +39,53 @@ observations:
       units: C
       min: -3.0
       max: 50.0
+- obs space:
```

Comment:
What's below is not tested, so remove for now. What's above does not really work yet. I'd suggest not touching this file for now.

Reply:
I can take it all out, but what do you mean "does not really work yet"?

Reply:
Related to what we talked about earlier, and somewhat "documented" here: https://github.com/NOAA-EMC/GDASApp/issues/869

Reply:
I removed the untested entries. Fixing the regexes will require more complicated testing and should be done later. Also, obsolete "output file" entries were removed (these are created automatically in the script)